### PR TITLE
stm32mp15 wifi fixes

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -477,6 +477,7 @@ PREFERRED_PROVIDER_virtual/bootloader:stm32mp1common ?= "u-boot-fio"
 PREFERRED_PROVIDER_u-boot:stm32mp1common ?= "u-boot-fio"
 PREFERRED_PROVIDER_virtual/trusted-firmware-a:stm32mp1common = "tf-a-fio"
 MACHINE_EXTRA_RRECOMMENDS:remove:stm32mp1common = "u-boot-fw-config-stm32mp"
+MACHINE_EXTRA_RRECOMMENDS:append:stm32mp1common = "linux-firmware-bcm43430"
```
Comment (quaresmajose) [inline meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:480]: misses the space `MACHINE_EXTRA_RRECOMMENDS:append:stm32mp1common = " linux-firmware-bcm43430"`

**Review by quaresmajose — APPROVED** [review]

**Comment by ricardosalveti** [comment]
Just missing the kernel bump.

**Review by MrCry0 — APPROVED** [review]
LGTM

**Comment by igoropaniuk** [comment]
@ricardosalveti done

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**